src/object-store/src/cache_policy.rs

```diff
@@ -122,7 +129,10 @@ impl<I: Accessor, C: Accessor> LayeredAccessor for LruCacheAccessor<I, C> {
                     Err(_) => return self.inner.read(&path, args).await.map(to_output_reader),
                 }
             }
-            Err(_) => return self.inner.read(&path, args).await.map(to_output_reader),
+            Err(err) => {
+                increment_counter!(OBJECT_STORE_LRU_CACHE_ERROR, OBJECT_STORE_LRU_CACHE_ERROR_KIND => format!("{}", err.kind()));
```

Comment:
The same as above suggestion.

Reply:
Done. See the [comment ](https://github.com/GreptimeTeam/greptimedb/pull/1405#discussion_r1169460752) below.